### PR TITLE
Send api calls through border http proxy to get user id

### PR DESCRIPTION
**Pull request by sczerwinski-wikia**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/core/Helios.java [hunk 1]
```diff
@@ -201,6 +214,26 @@ private static String getUserId(String userName) {
...
+    EnvType envType = Configuration.getEnvType(Configuration.getEnv());
+    File configFile = new File(Configuration.getCredentialsFilePath());
+
+    switch (envType) {
+      case DEV : {
+        return new HttpHost(
+            XMLReader.getValue(configFile, "border.poz.address"),
+            Integer.parseInt(XMLReader.getValue(configFile, "border.poz.port")),
+            XMLReader.getValue(configFile, "border.poz.protocol"));
+      } default: {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/Helios.java]: MAJOR Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
MAJOR Reduce this switch case number of lines from 6 to at most 5, for example by extracting code into methods. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)

File: src/test/java/com/wikia/webdriver/common/core/Helios.java [hunk 2]
```diff
@@ -201,6 +214,26 @@ private static String getUserId(String userName) {
     }
   }
 
+  private static HttpHost getBorderProxy() {
+
+    EnvType envType = Configuration.getEnvType(Configuration.getEnv());
+    File configFile = new File(Configuration.getCredentialsFilePath());
+
+    switch (envType) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/Helios.java]: MINOR Replace this "switch" statement by "if" statements to increase readability. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1301)

File: src/test/java/com/wikia/webdriver/common/core/Helios.java [hunk 3]
```diff
@@ -201,6 +214,26 @@ private static String getUserId(String userName) {
...
+      case DEV : {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/Helios.java]: MAJOR Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
MAJOR Reduce this switch case number of lines from 6 to at most 5, for example by extracting code into methods. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 4 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/common/core/Helios.java [hunk 4]
```diff
@@ -201,6 +214,26 @@ private static String getUserId(String userName) {
...
+        return new HttpHost(
+            XMLReader.getValue(configFile, "border.poz.address"),
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/common/core/Helios.java]: Why a default border for dev is POZ?

**Pull request merged**